.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -10,7 +10,7 @@ assignees: ""
 ### General information
 
   * OS: Linux / macOS / Windows
-  * Hypervisor: KVM / Hyper-V / hyperkit
+  * Hypervisor: KVM / Hyper-V / hyperkit / vfkit
```

Comment:
```diff
-  * Hypervisor: KVM / Hyper-V / hyperkit
+  * Hypervisor: KVM / Hyper-V / vfkit
```

Reply:
People might still report bugs against older releases, so I'd keep this for a while